.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Run Steampipe awstf compliance
+on: [pull_request,push]
+
+jobs:
+  steampipe-awstf-compliance:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Setup steampipe
+        id: setup-steampipe
+        uses: turbot/steampipe-action-setup@v1.4.0
+        with:
+          plugin-connections: |
+            connection "aws_tf" {
+              plugin = "terraform"
+              paths  = [ "./test-files/terraform/aws/**/*.tf" ]
+            }
+      - name: Steampipe AWS TF scan action
+        uses: ./
+        with:
+          mod-url: https://github.com/turbot/steampipe-mod-terraform-aws-compliance
+          additional-args: --output=none
```

Comment:
Why do we set these args?

Reply:
`--output` only affects the STDOUT, so this merely reduces the noise in the runner logs - made it easier to locate errors without scrolling during testing.

Would you prefer it be removed?

Reply:
Add a comment as to why we do this